index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <header class="header-section">
+        <div class="button-logo">
+            <a href="" id="logo"> My logo </a>
+        </div>
+<nav class="normal-button">
+    <img id="button" src="Icons/Vector.png" alt="button">
+</nav>
+    </header>
+    <section>
+        <p class="text">I’m Trevor Glad to see you!</p>
+        <p class="head-section">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+   <a href="" > 
+     <h2 class="lets">lets connect</h2>
+   </a> 
```

Comment:
- [ ] Why don't you use anchor tag itself for this link, let's use ` <a href="#" class="lets"> `
 ` lets connect`
   `</a> `